common/test/acceptance/tests/lms/test_lms_dashboard.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+"""
+End-to-end tests for the main LMS Dashboard (aka, Student Dashboard).
+"""
+from uuid import uuid4
+from ..helpers import UniqueCourseTest
+from ...fixtures.course import CourseFixture
+from ...pages.lms.auto_auth import AutoAuthPage
+from ...pages.lms.dashboard import DashboardPage
+from ...pages.lms.login_and_register import CombinedLoginAndRegisterPage
+
+
+class LmsDashboardPageTest(UniqueCourseTest):
+
+    def setUp(self):
+        """
+        Initializes the components (page objects, courses, users) for this test suite
+        """
+        # Some parameters are provided by the parent setUp() routine, such as the following:
+        # self.course_id, self.course_info, self.unique_id
+        super(LmsDashboardPageTest, self).setUp()
+
+        # Load page objects for use by the tests
+        self.dashboard_page = DashboardPage(self.browser)
+        self.login_page = CombinedLoginAndRegisterPage(
```

Comment:
Good catch -- it's a leftover from building up the new test case -- we're using AutoAuthPage below, instead.